.ci/azure/linux.yml

```diff
@@ -216,129 +229,180 @@ jobs:
+      # because of static build libgna is needed for python binary
+      export LD_LIBRARY_PATH=$(REPO_DIR)/temp/gna_03.00.00.1455.2/linux/x64:$LD_LIBRARY_PATH
+      python3 -m pytest -s $(INSTALL_TEST_DIR)/mo/unit_tests --junitxml=$(INSTALL_TEST_DIR)/TEST-ModelOptimizer.xml
     displayName: 'Model Optimizer UT'
     continueOnError: false
 
-  - script: . $(SETUPVARS) && $(INSTALL_TEST_DIR)/ov_core_unit_tests --gtest_print_time=1 --gtest_filter=-*IE_GPU* --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-NGraphUT.xml
+  - script: |
+      sudo apt-get install libtbb-dev libpugixml-dev -y
+      cmake --build $(BUILD_DIR) --target package -j8
+    condition: eq(variables['CMAKE_CPACK_GENERATOR'], 'DEB')
+    displayName: 'Build Debian packages'
+    continueOnError: false
+
+  - script: |
+      # install debian packages from previous release
+      wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+      sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+      sudo apt-get install --no-install-recommends gnupg -y
+      echo "deb https://apt.repos.intel.com/openvino/2022 focal main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
+      sudo apt-get update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/intel-openvino-2022.list
+      # sudo apt-get install openvino -y
+      # install our local one and make sure the conflicts are resolved
+      sudo apt-get install --no-install-recommends dpkg-dev -y
+      rm -r _CPack_Packages
+      dpkg-scanpackages . /dev/null | gzip -9c > Packages.gz
+      echo "deb [trusted=yes] file:$(BUILD_DIR) ./" | sudo tee /etc/apt/sources.list.d/openvino-local.list
+      sudo apt-get update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/openvino-local.list
+      sudo apt-get install openvino -y
+    workingDirectory: $(BUILD_DIR)
+    condition: eq(variables['CMAKE_CPACK_GENERATOR'], 'DEB')
+    displayName: 'Install Debian packages'
+    continueOnError: false
+
+  - script: cmake -DCMAKE_INSTALL_PREFIX=$(INSTALL_DIR) -P $(BUILD_DIR)/cmake_install.cmake
+    condition: ne(variables['CMAKE_CPACK_GENERATOR'], 'DEB')
+    displayName: 'Install openvino'
+    continueOnError: false
+
+  - script: ls -alR $(INSTALL_DIR)
+    condition: ne(variables['CMAKE_CPACK_GENERATOR'], 'DEB')
+    displayName: 'List install files'
+
+  - script: rm -fr $(BUILD_DIR)
+    displayName: 'Clean build dir'
+    continueOnError: false
+
+  - script: $(SAMPLES_INSTALL_DIR)/cpp/build_samples.sh -i $(INSTALL_DIR)
+    displayName: 'Build cpp samples'
+    continueOnError: false
+
+  - script: |
+      export CC=clang
+      export CXX=clang++
+      $(SAMPLES_INSTALL_DIR)/cpp/build_samples.sh -i $(INSTALL_DIR)
+    displayName: 'Build cpp samples - clang'
+    continueOnError: false
+
+  - script: $(SAMPLES_INSTALL_DIR)/c/build_samples.sh -i $(INSTALL_DIR)
```

Comment:
Should we add clang for c samples too?

Reply:
no, C API does not depend on compiler ABI